bedrock/events/models.py

```diff
@@ -14,40 +14,58 @@
 from bedrock.events.countries import country_to_continent
 
 
+def calendar_id_from_google_url(feed_url):
+    return feed_url.split('/')[5]
+
+
+def calendar_url_for_event(event, calendar_id):
+    template = 'https://www.google.com/calendar/embed?showCalendars=0&mode=week&wkst=1&' \
+               'src={calendar_id}&ctz=America/Los_Angeles&' \
```

Comment:
Autoescape in the template layer is escaping these. I think that's fine. Other option is to pre-escape them here and use `|safe` in the template.